IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml

```diff
@@ -4230,18 +4250,16 @@ components:
           description: This element can be used to pass specific information on the booking, such as a price difference with the initially offered price at booking time 
           
         provisionalBalance: 
-          $ref: '#/components/schemas/PriceDef'
```

Comment:
We must keep PriceDef for the fare part, as there might be a need to provide local currency and international currency for the same price. E.g. as local sales might have to use prices exactly approved by the goverment. The price in local currency can not be calculated from a EUR price used for foreign sales as there might occure rounding errors.

Reply:
In the fares element we still have PriceDef (I assumed it would be needed for compatibility with offline fares).
For other usages, I believe we should rely on the configuration of the API consumer and only return the agreed currency in a currencyPriceDef. If this is not correct we should discuss with the group because being multicurrency systematically seems to me like a significant increase in complexity. Please let me know, or add the topic on the review & comments page.

Reply:
ok, we need it for fares only